### PR TITLE
Updated Readme with steps to create and export github token

### PR DESCRIPTION
**Pull request by rish691**

### CODE REVIEW

File: tf-mnist/README.md [hunk 1]
```diff
@@ -76,7 +76,8 @@ If above commands succeeds, you are good to go !
         kubectl get pods -n kubeflow
 
 If there is any rate limit error from github, please follow the instructions at:
-[Github Token Setup](https://ksonnet.io/docs/tutorial#troubleshooting-github-rate-limiting-errors)
+1. Generate a new token.(https://blog.github.com/2013-05-16-personal-api-tokens)
```
Comment (amsaha) [inline tf-mnist/README.md]: I suggest you just add the information from https://github.com/ksonnet/ksonnet/blob/master/docs/troubleshooting.md#github-rate-limiting-errors and cut and paste it here.

Comment (ramdootp) [reply tf-mnist/README.md]: this looks already fixed by sreejith, can you sync your fork

Comment (rish691) [reply tf-mnist/README.md]: Yes it is already fixed. I will decline this pr.

**Comment by rish691** [comment]
Already fixed

**Pull request closed without merging**